apollo-router/src/plugins/telemetry/config_new/instruments.rs

```diff
@@ -100,8 +116,118 @@ impl InstrumentsConfig {
             .defaults_for_levels(self.default_requirement_level, TelemetryDataKind::Metrics);
     }
 
-    pub(crate) fn new_router_instruments(&self) -> RouterInstruments {
+    pub(crate) fn new_static_router_instruments(&self) -> HashMap<String, StaticInstrument> {
```

Comment:
Can we consider changing the term `static` to `built_in`? Static makes me think that the instruments have been changed are stored in a static somewhere.

Reply:
Fixed https://github.com/apollographql/router/pull/5629/commits/c4a8b14787030df5190bcecc59b4d53fdeb99dfb